docs/modelserving/v1beta1/amd/README.md

```diff
@@ -0,0 +1,189 @@
+# AMD Inference Server
+
+The [AMD Inference Server](https://xilinx.github.io/inference-server/main/index.html) is an easy-to-use inferencing solution specially designed for AMD CPUs, GPUs, and FPGAs.
+It can be deployed as a standalone executable or on a Kubernetes cluster with KServe or used to create custom applications by linking to its C++ API.
+This example demonstrates how to deploy a Tensorflow GraphDef model on KServe with the AMD Inference Server to run inference on AMD EPYC CPUs.
```

Comment:
would be nice to add reference link to `AMD EPYC CPU`

Reply:
Done